flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/Runner.java

```diff
@@ -70,7 +79,8 @@
  * <p>Algorithms must explicitly support each type of output via implementation of
  * interfaces. This is scalable as the number of outputs is small and finite.
  */
-public class Runner {
+public class Runner
+extends ParameterizedBase {
```

Comment:
`Parameter` / `SimpleParameter` are the interface / base class for parameters whereas `Parameterized` / `ParameterizedBase` are the interface / base class for components containing the parameters.